### PR TITLE
Trackers: news.kmib.co.kr

### PR DESCRIPTION
**Pull request by ghost**
### URL
http://news.kmib.co.kr/article/view.asp?arcid=0015623324&code=61121111

### Screenshot
Screenshot

### Network Logs
network1
network2
network3
network4

### CODE REVIEW

**Review by Alex-302 — CHANGES_REQUESTED** [review]
also maybe this can be blocked
`https://apis.naver.com/rmcnmv/rmcnmv/PlayCount.json?v=1&key=V122926442a1a2aa441324942***418db8b7874942abb4ce7e0336&tid=wzh14pAnjj***ChOZA&tcs=null&vid=A429220275F2E81DBAE58***F2AF5FAFB&cc=UA&sid=2010&inout=out`
Request made by player, and it is working when blocked

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -23,6 +23,11 @@
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 ||googleadservices.com/pagead/conversion_async.js$redirect=noopjs,domain=ffm.to,important
 !
+||js.ad4989.co.kr/common/js/tend.js
```
[the remark this answers is not in the document]
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt:26]: Sent you (`alex302@adguard.com`) the HAR file.

File: SpywareFilter/sections/specific.txt [hunk 2]
```diff
@@ -23,6 +23,11 @@
...
+||kmib.co.kr/NetInsight/trace/track
+||idm.skplanet.com/pixel
+||media.adpnut.com/cgi-bin/PelicanC.dll
```
[the remark this answers is not in the document]
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/77431/commits/e14ede0b456c0ae5ba5c57e51726b3ebf36674f3

File: SpywareFilter/sections/specific.txt [hunk 3]
```diff
@@ -23,6 +23,11 @@
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 ||googleadservices.com/pagead/conversion_async.js$redirect=noopjs,domain=ffm.to,important
 !
+||js.ad4989.co.kr/common/js/tend.js
+||kmib.co.kr/NetInsight/trace/track
```
[the remark this answers is not in the document]
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/77431/commits/e14ede0b456c0ae5ba5c57e51726b3ebf36674f3

[the remark this answers is not in the document]
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: Wait. I cannot find `/adsvr.` in AG Tracking Protection filter.
I want to add `||kmib.co.kr/NetInsight/trace/track` into AG Tracking Protection filter in order to block that without AG Base filter.

File: SpywareFilter/sections/specific.txt [hunk 4]
```diff
@@ -23,6 +23,9 @@
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 ||googleadservices.com/pagead/conversion_async.js$redirect=noopjs,domain=ffm.to,important
 !
+||js.ad4989.co.kr/common/js/tend.js
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:26]: It is blocked by Base filter

Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/77431/commits/0d50467d7f79bef13341fb9962f9654976087b7d

File: SpywareFilter/sections/specific.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:26]: Base filter is literally base)

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**